docs/userguide/configuring.rst

```diff
@@ -446,7 +446,16 @@ The following snippet shows an example configuration for accessing NSCC's **ASPI
 .. literalinclude:: ../../parsl/configs/ASPIRE1.py
 
 
+Blue Waters (NCSA)
```

Comment:
This section was removed a week or so back since this machine is now retired. I'd recommend removing changes to the `configuring.rst` file.